img/lms-ui-iconselectmenu.js

```diff
@@ -59,6 +59,9 @@ LmsUiIconSelectMenu.prototype.init = function() {
         .iconselectmenu()
         .iconselectmenu( 'menuWidget' );
 
+    // rewrite jQuery UI styles
+    $( this.select_id+"-button" ).addClass('lms-address-select');
+
```

Comment:
tak mi pasowało, lms-address-select = select ktory zawiera adresy w lms, klasa sama w sobie nic nie robi

p.s. oczywiscie jesli jest taka potrzeba to moge zmienic